release-notes/major9/09.19.0/valtimo-frontend-libraries.md

```diff
@@ -24,6 +24,11 @@ The following features were added:
 
   The `valtimo-modal` component now scrolls to the top when transitioning between form flow steps, and when switching pages in a Form.IO wizard.
 
+* **Added option to customize Left Sidebar width adjustment**
+
+  Added to the `valtimo-left-sidebar` component the options to customize the **default**, **maximum** and 
+  **minimum** width values of the menu, and the option to keep the resized value when refreshing the page.
```

Comment:
Is it an option to keep the resized value on refresh? I think it happens by default right, with no way to turn it off? Also, I would definitely mention how to customize the environment file here, this is essential for implementation developers.